### PR TITLE
Update Info.plist

### PR DESCRIPTION
**Pull request by DhairyaBahl**
This is the PR for the issue #1705 .

### CODE REVIEW

**Comment by DhairyaBahl** [comment]
@ice0 I made all the changes that were requested to solve the issue. Kindly look into it. 

Thanks :)

File: autobuild/osx/app-template/Contents/Info.plist [hunk 1]
```diff
@@ -4,9 +4,11 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(org.synfig.SynfigStudio)</string>
```
Comment (ice0) [inline autobuild/osx/app-template/Contents/Info.plist:8]: $(org.synfig.SynfigStudio)
should be without $(), like this
```
<string>org.synfig.sifz</string>
```

**Comment by ice0** [comment]
There is another one copyright string:
```
© 2001-2008 The Synfig Team
```

**Comment by DhairyaBahl** [comment]
@ice0 Okay I am working on it. :+1:

File: autobuild/osx/app-template/Contents/Info.plist (hunk 1 above)
Comment (DhairyaBahl) [reply autobuild/osx/app-template/Contents/Info.plist:8]: @ice0 Can you tell me how to rectify this without increasing number of commits :(

Thanks :)

**Comment by DhairyaBahl** [comment]
@ice0 I rectified it and also changed 2008 to 2020 in another line but there is one problem that i am facing, everytime i make changes in the file, it adds it as a new commit. What should I do ??

Kindly help

Thanks :)

**Comment by ice0** [comment]
> it adds it as a new commit. What should I do ??

Please read about `git squash`.

PR looks good for me, and I think this might be a good place here to learn how to squash commits.

**Review by ice0 — APPROVED** [review]

**Comment by DhairyaBahl** [comment]
@ice0 Yes, I am learning about squashing. Good to know it will solve this multiple commits problem.

Thanks :)

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**